cmd/krel/cmd/release_notes.go

```diff
@@ -308,6 +571,29 @@ func createDraftPR(tag string, result *releaseNotesResult) error {
 	return nil
 }
 
+// tryToFindPreviuousTag gets a release tag and returns the one before it
+func tryToFindPreviuousTag(tag string) (string, error) {
+	url, err := git.GetDefaultKubernetesRepoURL()
+	if err != nil {
+		return "", err
+	}
+
+	status, err := command.New(
+		"git", "ls-remote", "--sort=v:refname",
+		"--tags", url,
+	).
+		Pipe("grep", "-Eo", "v[0-9].[0-9]+.[0-9]+[-a-z0-9\\.]*$").
+		Pipe("grep", "-B1", tag).
+		Pipe("head", "-1").
+		RunSilentSuccessOutput()
+
+	if err != nil {
+		return "", err
+	}
+
+	return strings.TrimSpace(status.Output()), nil
+}
```

Comment:
This function should be part of the `git` package. I would also make it a bit more flexible by maybe introducing only a new `LSRemote()` function. 

(I think this can be done in a follow-up PR as well.)